luxonis_ml/nn_archive/config_building_blocks/base_models/input.py

```diff
@@ -65,6 +68,42 @@ class Input(CustomBaseModel):
         max_length=5,
         description="Shape of the input data as a list of integers (e.g. [H,W], [H,W,C], [BS,H,W,C], ...).",
     )
+    layout: str = Field(
+        description="Lettercode interpretation of the input layout (e.g., 'HCHW').",
```

Comment:
Typo, should be `'NCHW'`

Reply:
Fixed in [fdd6bff](https://github.com/luxonis/luxonis-ml/pull/109/commits/fdd6bff9eee194d210a767e258c1e47cd2f25986). Thanks!